common/src/main/java/org/apache/sedona/common/Functions.java

```diff
@@ -1045,4 +1045,18 @@ public static Double hausdorffDistance(Geometry g1, Geometry g2, double densityF
     public static Double hausdorffDistance(Geometry g1, Geometry g2) throws Exception{
         return GeomUtils.getHausdorffDistance(g1, g2, -1);
     }
+
+    public static int coordDim(Geometry geom) {
```

Comment:
No need to reimplement the function here. Just use nDim() func.

Reply:
Do you want me to remove the coordDim func entirely or have it, and call the nDims function in this file?

Reply:
@furqaankhan Please remove the coordDim func entirely. BTW, in the doc, please mention that this func is an alias of nDims.